utils/.gitignore

```diff
@@ -0,0 +1,3 @@
+release_notes.rst
+rs.cred
+fs.cred
```

Comment:
ok, license taken care of

On Mon, May 11, 2015 at 4:54 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In utils/.gitignore
> <https://github.com/cyclus/ciclus/pull/63#discussion_r30086646>:
>
> > @@ -0,0 +1,3 @@
> > +release_notes.rst
> > +rs.cred
> > +fs.cred
>
> Ahh cool.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/63/files#r30086646>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192